### PR TITLE
Feature landing

### PR DESCRIPTION
**Pull request by klapperkopp**
Adding a simple landing page that leads to the different views.

### CODE REVIEW

File: src/pages/IndexPage/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// @flow
+import React from "react";
+import clsx from "clsx";
+import config from "config";
+
+import useStyles from "./styles";
+
+import Button from "../../components/Button";
```
Comment (franziz) [inline src/pages/IndexPage/index.js]: I think we can use absolute import directly. Instead of using ../../ you can directly components/Button

Comment (klapperkopp) [reply src/pages/IndexPage/index.js]: yep, done.

**Review by klapperkopp — COMMENTED** [review]
@franziz can we merge this in?

**Comment by franziz** [comment]
Sorry, I missed this. 
Let me take a look first tomorrow, if everything good, I will merge

**Pull request merged**